pkgs/development/interpreters/python/cpython/default.nix

```diff
@@ -200,11 +209,11 @@ let
     [
       nukeReferences
     ]
-    ++ optionals (!stdenv.hostPlatform.isDarwin) [
+    ++ optionals (!stdenv.hostPlatform.isDarwin && !withMinimal) [
       autoconf-archive # needed for AX_CHECK_COMPILE_FLAG
       autoreconfHook
     ]
```

Comment:
Have you checked `python3Minimal` still builds on Darwin with this change? (I can test it if not.)